### PR TITLE
Add feedforward

### PR DESCRIPTION
**Pull request by McGiverGim**
This adds support for https://github.com/betaflight/betaflight/pull/6355

It is marked as `Don't merge` because this needs to be merged after https://github.com/betaflight/betaflight-configurator/pull/1119, and will need rebase and fix of the conflicts.

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -1224,22 +1224,22 @@
         "message": "Benewake LIDAR"
     },
     "pidTuningUpgradeFirmwareToChangePidController": {
-        "message": "<span class=\"message-negative\">Changing PID controller disabled - you can change it via the CLI.</span>  You have firmware with API version <span class=\"message-negative\">$1</span>, but this functionality requires <span class=\"message-positive\">$2</span>."
+        "message": "<span class=\"message-negative\">Changing PIDF controller disabled - you can change it via the CLI.</span>  You have firmware with API version <span class=\"message-negative\">$1</span>, but this functionality requires <span class=\"message-positive\">$2</span>."
```
Comment (mikeller) [inline locales/en/messages.json]: Not sure if we actually want to change how we call the controller in user facing text.
'PID controller' is a well established term (https://en.wikipedia.org/wiki/PID_controller), 'PIDF controller' is not. In fact, 'PIDF controller' seems to be used for a model that is 'PID + filter' (https://ieeexplore.ieee.org/document/7358695/), which is different from what we are doing.

Comment (McGiverGim) [reply locales/en/messages.json]: Yes, that was one of my doubts. This was discussed in slack and it seemed accepted, but if not, I can change it here and in the blackbox PR, both of them change the PID concept to PIDF concept.

**Comment by McGiverGim** [comment]
I think it's better to have a different PR to remove the unused PIDs. But if you prefer here I can add it.
I can't touch code until tomorrow so let me know the final decision.

**Comment by McGiverGim** [comment]
Renamed the PIDF to PID. Now this is waiting to https://github.com/betaflight/betaflight-configurator/pull/1119 to be merged.

**Comment by McGiverGim** [comment]
Rebased against master and ready to be merged...

**Pull request merged**